### DISCUSSION TITLE
Loot tracker - Reset other option request

### DISCUSSION THREAD

**Discussion by russel204** [runelite/runelite#14947] [Ideas]
Im requesting an option to "reset others" similar to how the exp trackers works. When there's only one option you want to save you can right click > reset others and everything else will be removed. It would be nice if something like this could be added to the loot tracker

**Comment by LRober09** [comment]
See related: https://github.com/runelite/runelite/discussions/15877

**Comment by LRober09** [comment]
PR posted: https://github.com/runelite/runelite/pull/15960
